lib/is-semantic-message.js

```diff
@@ -16,7 +16,7 @@ module.exports = function isSemanticMessage (message, validScopes, validTypes, a
   }
 
   const [result] = commits
-  const { scope, type } = result.header
-  const isScopeValid = !validScopes || !scope || validScopes.includes(scope)
+  const { scope: scopes, type } = result.header
+  const isScopeValid = !validScopes || !scopes || scopes.split(',').every(scope => validScopes.includes(scope))
```

Comment:
```suggestion
  const isScopeValid = !validScopes || !scopes || scopes.split(',').map(trim).every(scope => validScopes.includes(scope))
```

Reply:
☝🏼 In case people include spaces after their commas.

Reply:
Done and added test.